src/Lkl.cc

```diff
@@ -637,7 +646,7 @@ Double_t Lkl::CallMinimization(Double_t g,Bool_t isVerbose,Int_t strategy)
   arglist[0] = 10000;
   iflag = -1;
   Int_t counter = 0;
-  const Int_t maxcounts = 50;
+  const Int_t maxcounts = 300;
```

Comment:
where did you see this was necessary?

Reply:
Glory Duck sometimes need more than 150 iterations and I think a few times >200 was needed. I'm unsure why though... but the convergence is healthy, it just take a lot of iterations sometimes. Maybe in the future we can define a more clever maxcounts (based on TMinuit feedback?)?

Reply:
Ok